lua/barbecue/autocmd.lua

```diff
@@ -26,12 +26,17 @@ end
 ---Update winbar on necessary events.
 function M.create_updater()
   local events = {
-    "WinResized",
     "BufWinEnter",
     "CursorMoved",
     "InsertLeave",
   }
 
+  if vim.fn.has("nvim-0.9") == 1 then
```

Comment:
Please use this other function instead

```suggestion
  local v = vim.version()
  if v.major >= 0 and v.minor >= 9 then
```